src/app/pizza-toppings/pizza-toppings.component.html

```diff
@@ -33,6 +33,9 @@
         <br>
         <h3>
             Total: {{ totalPrice | currency }}
-        </h3>        
+        </h3>
     </mat-card-content>
-</mat-card>
\ No newline at end of file
+</mat-card>
+
+
+<app-pizza-calculator></app-pizza-calculator>
```

Comment:
I think you want to put this in app.component.html, not in the pizza-toppings components

Reply:
@tsteelematc 
done